README.md

```diff
@@ -0,0 +1,34 @@
+# XMTP React playground
+
```

Comment:
@nplasterer Do you think playgrounds need alpha/beta/production designations?

Reply:
I think playgrounds should be dev/local. We probably shouldnt be encouraging production usage in a playground.